app/src/main/java/com/example/busify/ui/passenger/PassengerListActivity.kt

```diff
@@ -21,11 +21,15 @@ class PassengerListActivity : AppCompatActivity() {
             PassengerRVModel("Passenger 4", false)
```

Comment:
@Adi8712 at line 17, should it be var or it should be removed?

Reply:
var should be removed

Reply:
using var works, it wasn't working then because I made a mistake in the initialisation of the adapter